test/cmdlineTests/~soljson_via_fuzzer/test.sh

```diff
@@ -10,7 +10,7 @@ cd "$SOLTMPDIR"
 "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/test/
 "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/docs/
 
-echo ./*.sol | xargs -P 4 -n 50 "${SOLIDITY_BUILD_DIR}/test/tools/solfuzzer" --quiet --input-files
-echo ./*.sol | xargs -P 4 -n 50 "${SOLIDITY_BUILD_DIR}/test/tools/solfuzzer" --without-optimizer --quiet --input-files
+grep -L --include="*.sol" "pragma experimental solidity;" * | xargs -P 4 -n 50 "${SOLIDITY_BUILD_DIR}/test/tools/solfuzzer" --quiet --input-files
```

Comment:
@cameel did it this way instead of the initial approach (see previous commit), since grep returns 1 if no matches, and no matches is exactly what I'm looking for, then xargs would throw 123 if the underlying command (grep) returns a non zero exit code. This works on Linux, but fails on mac; the weird thing is that I'm debugging on Mac at the moment, and it seems like it should be working - i.e. running `cmdlineTests.sh ~soljson_via_fuzzer` exits with 0, but it doesn't on the CI I guess? Do you have any ideas?

Reply:
> xargs would throw 123

You mean the first `xargs`? Why not skip it completely? You can just give grep the wildcard directly:

```bash
grep -L "pragma experimental solidity;" *.sol | ...
```

Reply:
GNU grep is not part of coreutils. Also, I tried it locally on my Mac machine with both BSD and GNU grep, and both worked fine, and returned 0, which is why I'm confused. We really should switch all of these to Python :)

Reply:
Ah true, I assumed it was but looks like it has its own `grep` package.

In any case, I'm getting 0 on my system too...